site-modules/userwrapper/manifests/user.pp

```diff
@@ -13,30 +14,30 @@
     ensure     => present,
     home       => $homedir,
     managehome => false,
-  } ->
+  }
 
-  file { "/var/users/${name}":
+  -> file { "/var/users/${name}":
```

Comment:
Do you know what this arrow does?

Reply:
@RandellP this is one of the chaining operators that puppet has.  They are used to declare that resources should be realized in a defined order.
[Ref](https://puppet.com/docs/puppet/latest/lang_relationships.html#reference-2122)